File: CaneOrbit.Api/Models/Propriedade.cs
namespace CaneOrbis.Api.Models
{
    public class Propriedade
    {
        public int IdPropriedade { get; set; }

        public int IdUsuario { get; set; }

        public string NmPropriedade { get; set; } = string.Empty;

        public string? DsLocalizacao { get; set; }

        public decimal? VlAreaHectare { get; set; }

        public Usuario? Usuario { get; set; }

        public ICollection<Field> Fields { get; set; } = new List<Field>();
    }
}

File: caneorbitJava/src/main/java/br/com/fiap/tdspo/gsolution/caneorbit/domain/model/Propriedade.java
package br.com.fiap.tdspo.gsolution.caneorbit.domain.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.util.ArrayList;
import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "T_ORB_PROPRIEDADE")
public class Propriedade {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ID_PROPRIEDADE", nullable = false)
    private Long id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "ID_USUARIO", nullable = false)
    private Usuario usuario;

    @Column(name = "NM_PROPRIEDADE", nullable = false, length = 100)
    private String nome;

    @Column(name = "DS_LOCALIZACAO", length = 150)
    private String localizacao;

    @Column(name = "VL_AREA_HECTARE", precision = 10, scale = 2)
    private BigDecimal areaHectare;

    @OneToMany(mappedBy = "propriedade", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
    private List<Field> fields = new ArrayList<>();
}